R/shiny_models_twocls.R

```diff
@@ -61,7 +70,17 @@ shiny_models.two_cls_shiny_data <-
       ),
       shinydashboard::dashboardBody(
         shinydashboard::tabItems(
-          # First tab content
+          # first tab content
+          shinydashboard::tabItem(
+            tabName = "tuning",
+            shiny::fluidRow(
+              boxed(
```

Comment:
I don't think that this needs to be in a box. It will probably be the only plot in this tab.